examples/configuration/src/index.ts

```diff
@@ -0,0 +1,24 @@
+import { DaprClient, CommunicationProtocolEnum } from "dapr-client";
```

Comment:
Have we kept this `CommunicationProtocolEnum` so that it's easier for users to add `GRPC` parameter to `DaprClient`? We might consider commenting it out. 

Similar for `daprAppId`.